wordpool/init_no_pandas.py

```diff
@@ -0,0 +1,28 @@
+def load_no_pandas(words_filepath):
```

Comment:
Rather than having public methods that have `no_pandas` in them, I would prefer if there was just a single function that could take a keyword argument to load without pandas. The logic of what's going on can be split into two private functions named, e.g. `_load_no_pandas` and `_load_pandas`.